packages/server/src/message/firebase-relay.ts

```diff
@@ -1,8 +1,15 @@
 import * as firebase from 'firebase';
 
-import * as fetch from 'node-fetch';
+import { Model } from 'objection';
 import '../../config/env';
+import { handleAppMessage } from '../app/handlers/handle-app-message';
+import { handleWalletMessage } from '../app/handlers/handle-wallet-message';
 import { HUB_ADDRESS } from '../constants';
+import knex from '../wallet/db/connection';
+
+// TODO: Currently the firebaseRelay is all that is needed to respond to the app
+// If the REST endpoint is not being used it should probably be removed
+Model.knex(knex);
```

Comment:
The firebase-relay shouldn't be responsible for setting up a db connection. As per our discussion, there should be a `hub` that has a connection, and the firebase relay can call `hub.handleWalletMessage`.